src/ClassFileLocator.php

```diff
@@ -51,7 +51,7 @@ public function __construct($dirOrIterator = '.')
      *
      * @return bool
      */
-    public function accept()
+    public function accept(): bool
```

Comment:
Declaring `: bool` here is a BC break, because the class is not `final`.

`#[\ReturnTypeWillChange]` is preferable in such a scenario.

Reply:
Alright, changed that.